<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyFields</class>
 <widget class="QDialog" name="MyFields">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="wid_fields">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Поле №1</string>
      </attribute>
      <widget class="QLabel" name="field1">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>641</width>
         <height>431</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>370</y>
         <width>181</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Будьте обережні!
Якщо ви підете додому, 
не зібравши урожай, 
то його з'їдять птахи!</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Виберіть, що бажаєте 
посадити:</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_carrot">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Морква</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_potato">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Картопля</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_onion">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Цибуля</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_cucumber">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Огірки</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_tomato">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Помідори</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_sunflower">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Соняхи</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>181</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Посадити</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>До збору 
урожаю менше:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_timer">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>секунд</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Стан поля:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_stan">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>250</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Пусто</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>280</y>
         <width>181</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Зібрати урожай</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Поле №2</string>
      </attribute>
      <widget class="QLabel" name="field2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>641</width>
         <height>431</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>370</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Будьте обережні!
Якщо ви підете додому, 
не зібравши урожай, 
то його з'їдять птахи!</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_potato_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Картопля</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>До збору 
урожаю менше:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>секунд</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>181</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Посадити</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_sunflower_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Соняхи</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_stan_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>250</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Пусто</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_carrot_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Морква</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Стан поля:</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_cucumber_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Огірки</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>280</y>
         <width>181</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Зібрати урожай</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Виберіть, що бажаєте 
посадити:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_timer_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_tomato_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Помідори</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_onion_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Цибуля</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Поле №3</string>
      </attribute>
      <widget class="QLabel" name="field3">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>641</width>
         <height>431</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>370</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Будьте обережні!
Якщо ви підете додому, 
не зібравши урожай, 
то його з'їдять птахи!</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_sunflower_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Соняхи</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_carrot_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Морква</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>230</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Стан поля:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_stan_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>250</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Пусто</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>181</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Посадити</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>280</y>
         <width>181</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Зібрати урожай</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>До збору 
урожаю менше:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_timer_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_onion_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Цибуля</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_tomato_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Помідори</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_cucumber_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Огірки</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb_potato_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Картопля</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Виберіть, що бажаєте 
посадити:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>секунд</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Повернутись додому</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
